targets/ath79-generic

```diff
@@ -76,3 +76,9 @@ device('ocedo-raccoon', 'ocedo_raccoon', {
 device('tp-link-archer-c6-v2', 'tplink_archer-c6-v2', {
 	packages = ATH10K_PACKAGES_QCA9888,
 })
+
+-- Ubiquity
+
+device('ubiquiti-nanostation-ac', 'ubnt_nanostation-ac', {
+	packages = ATH10K_PACKAGES_QCA9888,
+})
```

Comment:
With 64 MB RAM and one ath10k radio this device needs to be flagged as broken with reason "64M RAM ath9k + ath10k" .